src/risk_distributions/risk_distributions.py

```diff
@@ -492,22 +492,41 @@ def pdf(self, x: Union[pd.Series, np.ndarray, float, int]) -> Union[pd.Series, n
             p = p.values
         return p
 
-    def ppf(self, q: Union[pd.Series, np.ndarray, float, int]) -> Union[pd.Series, np.ndarray, float]:
+    def ppf(self, q: Union[pd.Series, np.ndarray, float, int],
+            q_dist: Union[pd.Series, np.ndarray, float, int]) -> Union[pd.Series, np.ndarray, float]:
+        """Quantile function using 2 propensities.
+
+        Parameters
+        ---------
+        q :
+            value propensity
+        q_dist :
+            propensity for picking the distribution
+
+        Returns
+        --------
+            Sample from the ensembled distribution.
+        """
         single_val = isinstance(q, (float, int))
```

Comment:
q_dist should have same type as q, should go through formatting

Reply:
Done.